doc/feature/compartmentReport.md

```diff
@@ -0,0 +1,422 @@
+Additions to compartmentReport to support ZeroEQ streams
+============
+
+## Motivation
+
+The current CompartmentReport lacks an implementation based on ZeroEQ as the
+SpikeReport does. There is a network plugin implementated with codash, but it
+has been deprecated. Since there's a new specification of the SpikeReport API
+ready for implementation, it makes sense to review the CompartmentReport
+interface to make it consistent with the new SpikeReport and prepare it for the
+ZeroEQ restrictions at the same time.
+
+This specification proposes a new interface for the CompartmentReport with
+these goals in mind.
+
+## API
```

Comment:
It depends on what type of spec and review you want to do. If you want to use this proposal to make  an incremental update on the current API it's better to list the differences, but it will make the document possibly harder to read. The document as is is easier to read with no prior knowledge.